utils/text_test.go

```diff
@@ -42,7 +42,10 @@ func TestTokenizeString(t *testing.T) {
 		{"math+=×÷√∊", []string{"math", "+", "=", "×", "÷", "√", "∊"}},     // math symbols treated as individual tokens
 		{"emoji😄🏥👪👰😟🧟", []string{"emoji", "😄", "🏥", "👪", "👰", "😟", "🧟"}},   // emojis treated as individual tokens
 		{"👍🏿 👨🏼", []string{"👍", "🏿", "👨", "🏼"}},                            // tone modifiers treated as individual tokens
-		{"ℹ︎ ℹ️", []string{"ℹ", "ℹ"}},                                      // variation selectors ignored
+		{"ℹ ℹ️", []string{"ℹ", "ℹ️"}},                                      // variation selectors ignored
```

Comment:
FWIW It seems the first i had some non printable character next to it, I had to remove that as well both here and in expressions

Reply:
What does this tokenize to? `หอบาเบลทั่วแผ่นดินโลกมีภาษาเดียวและมีสำเนียงเดียวกัน`

Reply:
that หอบาเบลทั่วแผ่นดินโลกมีภาษาเดียวและมีสำเนียงเดียวกัน

Reply:
Ya, so we kind of have to accept that we can't really tokenize these languages. Still think this PR is the right thing but we aren't really breaking up words with our tokenize here, rather just NOT breaking up individual characters. (that sentence above translates to "The Babel tower throughout the earth has one language and has the same accent.")

Reply:
Yep